modules/AWS/nfs/variables.tf

```diff
@@ -58,7 +52,7 @@ variable "shared_home_snapshot_id" {
     error_message = "Provide correct EBS snapshot ID."
   }
 }
-# The size of the EBS volume to allocate for shared home. If is null no PV and PVC will be created.
+
 variable "shared_home_size" {
```

Comment:
Please remove default value as this is not valid anymore. Also add validation for shared_home please.